android/engine/src/main/java/org/smartregister/fhircore/engine/configuration/app/ApplicationConfiguration.kt

```diff
@@ -33,5 +33,6 @@ data class ApplicationConfiguration(
   val syncStrategies: List<String> = listOf(),
   val loginConfig: LoginConfig = LoginConfig(),
   val deviceToDeviceSync: DeviceToDeviceSyncConfig? = null,
-  val snackBarTheme: SnackBarThemeConfig = SnackBarThemeConfig()
+  val snackBarTheme: SnackBarThemeConfig = SnackBarThemeConfig(),
+  val periodicTaskUpdateInterval: Long = 1
```

Comment:
Consult with @ahsanbhatti49 on the changes requested on this config property. The ideas is to use[ ISO period format ](https://en.wikipedia.org/wiki/ISO_8601#:~:text=ISO%208601%20uses%20the%2024,hour%20between%2000%20and%2024.) to have the flexibility of defining duration

Reply:
We can parse the string using [java.time.Duration#parse](https://developer.android.com/reference/java/time/Duration#parse(java.lang.CharSequence))